examples/rust/udp-push-pop.rs

```diff
@@ -91,7 +91,8 @@ impl UdpServer {
     }
 
     fn run(&mut self, local: SocketAddr, fill_char: u8, num_sends: usize) -> Result<()> {
-        let num_receives: usize = (8 * num_sends) / 128;
+        // Assume that if we send twice as many packets, most of them will get through.
```

Comment:
Do you mean to say that if we send out `half` the number of packets, we can assume all will get through?

Reply:
Sorry, we send out num_send and then expect half of of them to be received. I'll clarify.